site/content/docs/5.3/utilities/colors.md

```diff
@@ -46,16 +53,72 @@ Color utilities like `.text-*` that generated from our original `$theme-colors`
 **Deprecation:** With the addition of the expanded theme colors and variables, the `.text-muted` utility has been deprecated as of v5.3.0. Its default value has also been reassigned to the new `--bs-secondary-color` CSS variable to better support color modes. It will be removed in v6.0.0.
 {{< /callout >}}
 
-{{< callout info >}}
-For each `.text-*` there is a matching `.text-*-emphasis` utility. In Boosted, they have exactly the same value so we decided not to display them in the example above so that you don't hesitate on which class to use.
+<!-- Boosted mod -->
+## Orange's colors
 
-Here is a list of these extra classes:
-{{< colors.inline >}}
-{{- range (index $.Site.Data "theme-colors") }}
-- `.text-{{ .name }}-emphasis`
-{{- end -}}
-{{< /colors.inline >}}
-{{< /callout >}}
+Bootstrap provides many `.text-*` and `.bg-*` utilities, but they should be used with care to meet our design specifications and [WCAG 2.0 accessibility standards for color contrast](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html).
+To be sure to respect the specifications, it is necessary to define `color`, `background-color` and `font-size` altogether.
+
+Thus, the `.text-primary` color on white background (`#f16e00`) can only be used in a font size greater than 24px (using for example `.fs-3` utility), or 19px bold (using for example `.fs-4` and `.fw-bold` utilities).
+The `.text-primary` color on dark background (`#ff7900`) can be used in any size, and it shouldn't be used on light grey backgrounds at all.
+
+Here are some compliant combinations examples for texts:
+
+{{< example >}}
+<p class="text-primary fs-3">.text-primary</p>
+<p class="text-primary fs-4 fw-bold">.text-primary</p>
+<p class="text-primary bg-dark">.text-primary</p>
+<p class="text-secondary">.text-secondary</p>
+<p class="text-light bg-dark">.text-light</p>
+<p class="text-body">.text-body</p>
+<p class="text-body-secondary">.text-body-secondary</p>
+<p class="text-black">.text-black</p>
+<p class="text-white bg-dark">.text-white</p>
+{{< /example >}}
+
+Here are some compliant combinations examples for non-texts elements, such as SVG icons:
+
+{{< example >}}
+<p class="p-2">
+  <svg width="1.875em" height="1.875em" class="text-primary" aria-hidden="true" focusable="false">
+    <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#info"/>
```

Comment:
Just wondering here if we shouldn't use other icons associated with the primary and secondary colors. The info icon is generally associated with the "info" color, isn't it?

Reply:
I changed the info icons with "settings", but the size is not exactly the same since we don't have the same viewbox, which was the start of this series of PR...

Reply:
IT will be corrected with PR #1929